airflow/providers/amazon/aws/utils/sqs.py

```diff
@@ -0,0 +1,90 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+import json
+import logging
+from typing import Any, Literal
+
+from jsonpath_ng import parse
+
+log = logging.getLogger(__name__)
+
+
+def process_response(
+    response: Any,
+    message_filtering: Literal["literal", "jsonpath"] | None = None,
+    message_filtering_match_values: Any = None,
+    message_filtering_config: Any = None,
+) -> Any:
+    """
+    Process the response from SQS.
+
+    :param response: The response from SQS
+    :return: The processed response
+    """
+    if "Messages" not in response:
```

Comment:
As `response` is `Any`, should we check whether it's iterable? Or should we change `response` to another type